src/main/java/pl/cleankod/exchange/core/mapper/AccountMapper.java

```diff
@@ -0,0 +1,15 @@
+package pl.cleankod.exchange.core.mapper;
+
+import pl.cleankod.exchange.core.domain.Account;
+import pl.cleankod.exchange.core.domain.Money;
+import pl.cleankod.exchange.core.dto.AccountDto;
+import pl.cleankod.exchange.core.dto.MoneyDto;
+
+public class AccountMapper {
+
+    public AccountDto toDto(final Account entity, final Money money) {
```

Comment:
What if the DTOs could create themselves from the domain objects? Then we won't need additional classes to do so. What do you think?